account_invoice_start_end_dates/models/account_invoice.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2013-2016 Akretion, Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError, UserError
+
+
+class AccountInvoiceLine(models.Model):
+    _inherit = 'account.invoice.line'
+
+    start_date = fields.Date('Start Date')
+    end_date = fields.Date('End Date')
```

Comment:
The string values can be removed, because Odoo will create the same strings as declared automatically. Cool, or? : )

Reply:
I guess it would, but I prefer keeping it explicit, just for clarity

Reply:
It is required for a good style. You can check here - https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#38fields